sdk/src/main/java/com/horizen/account/state/EvmMessageProcessor.java

```diff
@@ -59,6 +59,7 @@ public byte[] process(Message msg, BaseAccountStateView view, GasPool gas) throw
                     // returnData here will include the revert reason
                     throw new ExecutionRevertedException(returnData);
                 case "out of gas":
+                case "contract creation code storage out of gas":
```

Comment:
1) Are we sure now if we cover all the "out of gas" errors of EVM?
2) Relying on error text description seems to be very-very unstable.

There is a list of EVM errors in `core/vm/errors.go`.
We need to introduce mapping between that go types and java wrapper types.
Then here we may do a patter matching per type instead of per text.
@JcBernack, want to know your opinion as well.

Reply:
Agree with you.
This way of error handling is much better, and `reverted` flag is much clear for external user (like EvmMsgProcessor).
@akrckn let's proceed in this way in current PR.